contracts/JBETHPaymentTerminal.sol

```diff
@@ -64,25 +64,28 @@ contract JBETHPaymentTerminal is JBPayoutRedemptionPaymentTerminal {
     @notice
     Transfers tokens.
 
-    ignored: _from The address from which the transfer should originate.
+    @param _from The address from which the transfer should originate.
     @param _to The address to which the transfer should go.
     @param _amount The amount of the transfer, as a fixed point number with the same number of decimals as this terminal.
   */
   function _transferFrom(
-    address,
+    address _from,
     address payable _to,
     uint256 _amount
   ) internal override {
+    _from; // Prevents unused var compiler and natspec complaints.
```

Comment:
\o/

Reply:
<img width="400" alt="image" src="https://user-images.githubusercontent.com/77952627/166123456-284b5037-b987-4385-8bf7-4d1cea5e6c6e.png">
makes this much better ^^